modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/AndesConfigurationManager.java

```diff
@@ -168,6 +169,11 @@ public static void initialize(int portOffset) throws AndesException {
             // set carbonPortOffset coming from carbon
             AndesConfigurationManager.carbonPortOffset = portOffset;
 
+            //set delivery timeout for a message
+            int deliveryTimeoutForMessage = AndesConfigurationManager.readValue(AndesConfiguration
+                    .PERFORMANCE_TUNING_TOPIC_MESSAGE_DELIVERY_TIMEOUT);
+            AndesContext.getInstance().setDeliveryTimeoutForMessage(deliveryTimeoutForMessage);
```

Comment:
What i meant was, we can set this property at the AndesContext static constructor right ? Is the concern that AndesConfiguration won't be ready by the time AndesContext constructor is fired ? 

(Idea is to read properties from memory outside this class, since this method (AndesConfigurationManager.initialize()) primarily reads the configs from file to memory and does any initial adjustments to prepare them for reading from outside.)